docs/supported_apis/series_supported.rst

```diff
@@ -463,6 +476,7 @@ the related section on :doc:`Defaulting to pandas </supported_apis/index>`.
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``value_counts``            | Y                               | The indices order of resulting object may differ   |
 |                             |                                 | from pandas.                                       |
+|                             |                                 | **Omnisci**:``Y``?                                 |

```

Comment:
`value_counts` should be fully supported for except of `dropna=True` parameter

Reply:
Right, I see this in tests. Fixed